ui-v2/tests/pages/dc/services/instance.js

```diff
@@ -17,6 +17,6 @@ export default function(visitable, attribute, collection, text, tabs) {
     addresses: collection('#addresses [data-test-tabular-row]', {
       address: text('[data-test-address]'),
     }),
-    metadata: collection('#metadata [data-test-tabular-row]', {}),
+    meta: collection('.meta [data-test-tabular-row]', {}),
```

Comment:
Just the `meta` > `metadata` thing again here, I'm trying to train my fingers to never write just `meta` again also! 😂